lib/commands/app/rename.ts

```diff
@@ -120,7 +120,9 @@ export class FleetRenameCmd extends Command {
 		} catch (e) {
 			// BalenaRequestError: Request error: "organization" and "app_name" must be unique.
 			if ((e.message || '').toLowerCase().includes('unique')) {
-				throw new ExpectedError(`Error: fleet ${params.fleet} already exists.`);
+				throw new ExpectedError(
+					`Error: fleet ${params.newName} already exists.`,
```

Comment:
```suggestion
					`Error: fleet ${newName} already exists.`,
```
If I am reading it right, `params.newName` may be `undefined` in which case `newName` (rather than `params.newName`) will store the new name interactively typed by the user (see definition of `const newName =` in line 108).

Reply:
Thanks for reviewing, @pdcastro ! I'll amend the commit to apply that change.